reflex/components/forms/debounce.py

```diff
@@ -78,3 +79,29 @@ def props_not_none(c: Component) -> dict[str, Any]:
     """
     cdict = {a: getattr(c, a) for a in c.get_props() if getattr(c, a, None) is not None}
     return cdict
+
+
+def _collect_first_child_and_props(c: Component) -> tuple[Component, dict[str, Any]]:
```

Comment:
I think we need to use `Tuple` for Python 3.7 support

Reply:
this file uses `from __future__ import annotations`, which enables PEP 563 behavior (even in 3.7), allowing these annotations to work.

i can change them if it matters, but the tests are passing on 3.7 and AFAIK `typing.Tuple` and `typing.Dict` are on their way out once 3.9 goes EoL, so if we can support the newer style now, with no drawbacks, i think that reduces our future burden to fix.

Reply:
Ah okay got it, we should convert all the code to use this then it's much nicer